libs/designsystem/src/lib/scss/base/_functions.scss

```
@@ -101,6 +101,11 @@
   @return $classes;
 }
 
+// Strips the unit by dividing it with exactly one of that unit, eg. 42px / 1px = 42
+@function unitless($number) {
+  @return $number / ($number * 0 + 1);
+}
```

We need to rename the function. Sass has a built-in function named `unitless()` and we're not using Sass modules and namespaces (yet).

The function also needs to a bit more robust. Consider applying the suggested change:
```suggestion
/**
 * Remove the unit of a length
 * @param {Number} $number - Number to remove unit from
 * @return {Number} - Unitless number
 * 
 * Source: https://css-tricks.com/snippets/sass/strip-unit-function/
 */
@function strip-unit($number) {
  @if type-of($number) == 'number' and not unitless($number) {
    @return $number / ($number * 0 + 1);
  }

  @return $number;
}
```